riptide-spring-boot-autoconfigure/README.md

```diff
@@ -236,6 +236,17 @@ Required when `propagate-flow-id` is enabled.
 </dependency>
 ```
 
+#### `Telemetry`
```

Comment:
```suggestion
#### Telemetry
```

Reply:
Should we rename the section in line 210 from `Tracing` to `OpenTracing` ? Just to separate one from the other, as they are both related to Tracing.